public/scripts/slash-commands.js

```diff
@@ -323,6 +324,29 @@ SlashCommandParser.addCommandObject(SlashCommand.fromProps({
     helpString: 'Opens up a chat with the character or group by its name',
     aliases: ['char'],
 }));
+SlashCommandParser.addCommandObject(SlashCommand.fromProps({
+    name: 'rename-char',
+    /** @param {{silent: string, renameChats: string}} options @param {string} name */
+    callback: async ({ silent = 'true', renameChats = null }, name) => {
+        const renamed = await renameCharacter(name, { silent: silent === 'true', renameChats: renameChats !== null ? renameChats === 'true' : null });
+        return String(renamed);
+    },
+    returns: 'true/false - Whether the rename was successful',
```

Comment:
AFAIK `returns` is intended to use with `ARGUMENT_TYPE`, not plain text

Reply:
I asked Len this when I made the tag commands, and he said its the opposite. He wanted to have type hinting for it too, but didn't have time before the merge.

It's just plain text as an explanation what is returned, and that's how it is used in nearly all other slash commands in the code right now.